delete-user-data/extension.yaml

```diff
@@ -99,6 +102,20 @@ params:
       For example, if you have the collections `users` and `admins`, and each collection
       has documents with User ID as document IDs, then you can enter `users/{UID},admins/{UID}`.
 
+  - param: FIRESTORE_DELETE_MODE
+    type: select
+    label: Cloud Firestore delete mode
+    description: >-
+      How would you like Firestore to delete documents? Set to `recursive` to also delete nested Collections and
+      Subcollections.
+    options:
+      - label: Shallow
+        value: shallow
+      - label: Recursive
+        value: recursive
+    default: shallow
```

Comment:
should probably default to recursive, since I would imagine that in almost all cases, nested collections would also contain user data that should be deleted.

Reply:
I can change this, however it'll be a breaking change for users who update (as it's currently shallow). If that's not an issue I'll change it 😊

Reply:
That's not a big concern since when they update, they'll be prompted to pick a value for FIRESTORE_DELETE_MODE (it won't silently set it to recursive). But yes it does technically qualify as a breaking change. This reminds me, can you update the version field in this PR also? (To 0.2.0 to indicate the breaking change).

Reply:
After internal discussions, we decided to keep it as shallow.